exe/vernier

```diff
@@ -38,6 +47,11 @@ FLAGS:
         o.on('--hooks [HOOKS]', String, "enable instrumentation hooks, currently supported: rails") do |s|
           options[:hooks] = s
         end
+        o.on("--metadata KEY=VALUE", String, "Set metadata key-value pairs (can be specified multiple times)") do |kv|
+          key, value = kv.split('=')
+          options[:metadata] ||= Metadata.new
+          options[:metadata] << { key: key, value: value }
```

Comment:
Do these entries need to be a hash at this point? Can we not just use an array tuple?

```suggestion
          options[:metadata] << [key, value]
```